example/server.js

```diff
@@ -47,5 +45,11 @@ server.get('/metrics/counter', function(req, res) {
 	res.end(register.getSingleMetricAsString('test_counter'));
 });
 
+server.get('/metrics/protobuf', function(req, res) {
+	client.collectDefaultMetrics();
```

Comment:
@SimenB i'm assuming you mean the `client.collectDefaultMetrics();` part, not the `server.get('/metrics/protobuf', function(req, res) {` part.

will do.